### ISSUE TITLE
Python 3 compatibility

### ISSUE THREAD

**Issue by zupo** [domenkozar/nose-selecttests#7]
Fails in Python 3 environment with:

```
eggs/nose-1.3.7-py3.5.egg/nose/plugins/manager.py:395: RuntimeWarning: Unable to load plugin selecttests = noseselecttests:NoseSelectPlugin: cannot import name 'ClassType'
```

**Comment by domenkozar** [comment]
I've pushed Python 3 branch, feel free to test it: https://github.com/domenkozar/nose-selecttests/commit/9098fa4d3bb0eb73d91b4e6631e630cd7977a6a6

**Comment by zupo** [comment]
LGTM

**Comment by domenkozar** [comment]
Closed via e75ab8d43c627d7daa55ca19b258e80059374bb7
